packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -27,6 +27,7 @@ export interface Conformant {
   exportedAtTopLevel?: boolean
   rendersPortal?: boolean
   wrapperComponent?: React.ReactType
+  handlesAsProp?: boolean
```

Comment:
As the Animation is not rendering root element, but only copies the children, we don't have the "as" prop there.